tests_thien/pages/dashboard/login_page.py

```diff
@@ -0,0 +1,31 @@
+from selenpy.element.text_box import TextBox
+from selenpy.support.conditions import have
+from selenpy.support import browser
+from selenpy.element.base_element import BaseElement
+from selenpy.element.combo_box import ComboBox
+from tests_thien.utilities import constant, browser_helper
+
+class LoginPage():
+  
+    def __init__(self):
+        self._cbb_repository = ComboBox("name=repository")
+        self._txt_username = TextBox("id=username")
+        self._txt_password = TextBox("id=password")
+        self._btn_login = BaseElement("class=btn-login")
+    
+    def open(self):
+        browser.open_url(constant.DA_URL)
+        browser.wait_until(have.title(constant.DA_TITLE))
+        
+    def login(self,username,password,repository=constant.DA_REPO):
+        self._cbb_repository.select_by_visible_text(repository)
+        self._txt_username.clear()
+        self._txt_username.send_keys(username)
+        self._txt_password.send_keys(password)
+        self._btn_login.click()
+        
+    def is_message_displayed(self,messge):
+        return browser_helper.get_alert_text() == messge
+        
+    def close_alert(self):
+        browser_helper.dismiss_alert()
```

Comment:
Remove it. You have dismiss_alert() in browser_helper already.

Reply:
removed